Defs/Ammo/HighCaliber/338LapuaMagnum.xml

```diff
@@ -132,7 +132,7 @@
 		<label>.338 Lapua Magnum bullet (FMJ)</label>
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageAmountBase>26</damageAmountBase>
-			<armorPenetrationSharp>11</armorPenetrationSharp>
```

Comment:
Unless you've got a compelling source to cite, leave it--from the good sources I've got and some napkin math, we're already being _very_ generous w/ .338 LM FMJ's sharp pen.

Reply:
Yea there is real world testing done and .338 Lapua penetrates half an inch of mild steel plate (commonly used for CE's AP values), linked in the spreadsheet. Can lower to 11.5 instead or nerf .338 Norma by 0.5 as it's a heavier and slower round delivering less energy overall.